website/docs/language/settings/backends/cos.mdx

```diff
@@ -9,7 +9,10 @@ description: >-
 
 Stores the state as an object in a configurable prefix in a given bucket on [Tencent Cloud Object Storage](https://intl.cloud.tencent.com/product/cos) (COS).
 
-This backend supports [state locking](/terraform/language/state/locking).
+This backend supports [state locking](/terraform/language/state/locking). You need following permissions for your credentials:
+
+* `CreateTag`, `DeleteTag` and `DescribeTags` on the tag key `tencentcloud-terraform-lock`.
```

Comment:
Small suggestion for using `-` for markdown lists, removing the final period on the sentence fragment, and adding an oxford comma:
```suggestion
- `CreateTag`, `DeleteTag`, and `DescribeTags` on the tag key `tencentcloud-terraform-lock`
```